engine/src/test/scala/cromwell/webservice/routes/wes/ServiceInfoSpec.scala

```diff
@@ -3,7 +3,8 @@ package cromwell.webservice.routes.wes
 import akka.actor.Props
 import akka.http.scaladsl.testkit.ScalatestRouteTest
 import akka.util.Timeout
-import cromwell.languages.config.{CromwellLanguages, LanguageConfiguration}
+import cromwell.languages.config.CromwellLanguages
+import cromwell.languages.config.LanguageConfiguration
```

Comment:
Would you mind backing out these changes since we didn't otherwise change the file?